addons/maptools/functions/fnc_mapToolDrawRadiusMarker.sqf

```diff
@@ -0,0 +1,61 @@
+#include "script_component.hpp"
+/*
+ * Author: Seb
+ * Creates a display that allows a maptool user to mark a radius of x around the 0 point of their maptool
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * call ace_maptools_fnc_mapToolDrawRadiusMarker
+ *
+ * Public: No
+ */
+
+if (GVAR(mapTool_Shown) isEqualTo 0) exitWith {};
+[{
+    private _display = findDisplay 12 createDisplay QGVAR(drawRadiusDisplay);
+    private _textCtrl = _display displayCtrl 451;
+    _textCtrl ctrlSetText "100";
+    _display displayAddEventHandler ["Unload", {
+        params ["_display", "_exitCode"];
+        if !(_exitCode isEqualTo 1) exitWith {};
```

Comment:
Since 'Arma 3 2.02' the `!(... isEqualTo ...)` [can be replaced by](https://community.bistudio.com/wiki/isNotEqualTo) `(... isNotEqualTo ...)`.

Reply:
Exitcode of 1 specifically means the cancel button was pressed